python/ray/tune/tune.py

```diff
@@ -78,6 +78,7 @@ def run_experiments(experiments, scheduler=None, **ray_args):
         print(runner.debug_string())
 
     for trial in runner.get_trials():
+        # TODO(rliaw): What about errored?
```

Comment:
That should be covered, since ERROR != TERMINATED

Reply:
The interaction here is a little odd because after executing the entire experiment, you get an exception that says "Trial did not complete!"

Reply:
Yeah that's the intent...